src/config.ts

```diff
@@ -1,33 +1,52 @@
+    intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES, Intents.FLAGS.GUILD_EMOJIS_AND_STICKERS, Intents.FLAGS.GUILD_VOICE_STATES],
+    makeCache: Options.cacheWithLimits({
+        MessageManager: {
+            maxSize: Infinity,
+            sweepInterval: 300,
+            sweepFilter: LimitedCollection.filterByLifetime({
+                lifetime: 10800
+            })
+        },
+        ThreadManager: {
+            maxSize: Infinity,
+            sweepInterval: 300,
+            sweepFilter: LimitedCollection.filterByLifetime({
+                lifetime: 10800,
+                getComparisonTimestamp: e => e.archiveTimestamp!,
+                excludeFromSweep: e => !e.archived
+            })
+        }
+    }),
+    retryLimit: 3
 };
+export const defaultPrefix = process.env.CONFIG_PREFIX?.replace(/"/g, "") ?? "!";
 export const owners: string[] = process.env.CONFIG_OWNERS?.replace(/  +/g, " ").split(/,[ ]?/) ?? [];
-export const YouTubeDataRetrievingStrategy = process.env.CONFIG_YOUTUBE_DATA_STRATEGY?.toLowerCase() as ("scrape" | "api" | undefined) ?? "scrape";
-export const disableInviteCmd = process.env.CONFIG_DISABLE_INVITE_CMD?.toLowerCase() === "yes";
-export const defaultVolume = Number(process.env.CONFIG_DEFAULT_VOLUME) || 50;
-export const maxVolume = Number(process.env.CONFIG_MAX_VOLUME) || 100;
-export const disableSongSelection = process.env.CONFIG_DISABLE_SONG_SELECTION?.toLowerCase() === "yes";
-export const searchMaxResults = Number(process.env.CONFIG_SEARCH_MAX_RESULTS) || 10;
-export const selectTimeout = Number(process.env.CONFIG_SELECT_TIMEOUT) * 1000 || 20 * 1000;
-export const allowDuplicate: boolean = process.env.CONFIG_ALLOW_DUPLICATE?.toLowerCase() === "yes";
-export const cacheYoutubeDownloads: boolean = process.env.CONFIG_CACHE_YOUTUBE_DOWNLOADS?.toLowerCase() === "yes";
-export const cacheMaxLengthAllowed = Number(process.env.CONFIG_CACHE_MAX_LENGTH) || 5400;
-export const deleteQueueTimeout = Number(process.env.CONFIG_DELETE_QUEUE_TIMEOUT) * 1000 || 180 * 1000;
-export const totalShards: string | number = process.env.CONFIG_TOTALSHARDS?.toLowerCase() ?? "auto";
-export const fetchAllUsers = process.env.CONFIG_FETCH_ALL_USERS?.toLowerCase() === "yes";
-export const debug = process.env.CONFIG_DEBUG?.toLowerCase() === "yes";
-
-if (searchMaxResults < 1) throw new Error("CONFIG_SEARCH_MAX_RESULTS cannot be smaller than 1");
-if (searchMaxResults > 10) throw new Error("CONFIG_SEARCH_MAX_RESULTS cannot be higher than 10");
-if (totalShards !== "auto" && isNaN(totalShards as unknown as number)) throw new Error("CONFIG_TOTALSHARDS must be a number or \"auto\"");
-if (!["scrape", "api"].includes(YouTubeDataRetrievingStrategy)) throw new Error("CONFIG_YOUTUBE_DATA_STRATEGY must be scrape or api");
+export const embedColor = process.env.CONFIG_EMBED_COLOR?.toUpperCase() ?? "3CAAFF";
+export const devGuild = JSON.parse(process.env.CONFIG_DEV_GUILD!) ?? [];
```

Comment:
```suggestion
export const devGuild = JSON.parse(process.env.CONFIG_DEV_GUILD! || "[]");
```
`JSON.parse` will throwing error when parsing undefined value